locale/en_US/manager.po

```diff
@@ -808,11 +808,28 @@ msgstr "Items with DOIs"
 msgid "doi.manager.settings.enableFor"
 msgstr "{$objects} will be assigned a DOI"
 
+msgid "doi.manager.settings.doiPrefix.required"
+msgstr "A DOI prefix is required"
+
+
 msgid "doi.manager.settings.doiSuffix"
 msgstr "DOI Suffix"
 
+msgid "doi.manager.settings.doiSuffix.default.description"
+msgstr "Use the default DOI suffix generation?"
+
+msgid "doi.manager.settings.doiSuffix.pattern"
+msgstr "DOI Suffix Pattern"
+
+msgid "doi.manager.settings.doiSuffix.custom"
+msgstr "Custom Pattern"
+
+msgid "doi.manager.settings.doiSuffix.customSubmission"
+msgstr "Custom Submission Pattern"
```

Comment:
This and the other labels can just be "Submissions", "Galleys", "Issues". The group should give adequate explanation.